{{ .ProjectSnake }}/requirements/runtime.txt

```diff
@@ -1,9 +1,7 @@
-#
-# This file is autogenerated by pip-compile with Python 3.12
-# by the following command:
-#
-#    bazel run //requirements:runtime.update
-#
+# This file was autogenerated by uv via the following command:
+#    bazel run @@//requirements:runtime
+--index-url https://pypi.org/simple
```

Comment:
Is this supposed to be on its own non-comment line?

Reply:
yes, that's right. `uv` is echo'ing one of the command-line arguments it recieved, so this file will replay correctly if you pass it to `pip install`

Amusingly the format of the requirements file is essentially "whatever pip install accepts"